src/ui/manage-ui/src/FamilyHubs.ServiceDirectory.Admin.Web/Pages/manage-services/delete-service.cshtml

```diff
@@ -22,6 +23,16 @@
                 </h1>
             </legend>
             <p class="govuk-body">By deleting this service you will completely remove it from both the directory and Manage family support services and accounts.</p>
+            @if (Model.UserRoleCanSeeConnectionRequestWarning)
+            {
+                <div class="govuk-warning-text">
+                    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+                    <strong class="govuk-warning-text__text">
+                        <span class="govuk-visually-hidden">Warning</span>
+                        You cannot delete a service if it has connection requests that have not been actioned.
+                    </strong>
+                </div>
+            }
```

Comment:
Question: can a user who doesn't have access to delete a service reach this page? If so, is there any use for if-check? I.e. should this warning be shown to everyone who has access to reach this page?

Reply:
Me & Andy discussed this with Casey as we had the same Q, but he confirmed it should be only VCS managers / dual roles that see this warning